modules/integration/src/test/scala/scala/cli/integration/BloopTests.scala

```diff
@@ -1,6 +1,7 @@
 package scala.cli.integration
 
 import com.eed3si9n.expecty.Expecty.expect
+import os.Pipe
```

Comment:
NIT don't import stuff from the `os.*` package, we generally use things from that package directly